librosa/sequence.py

```diff
@@ -112,6 +116,10 @@ def dtw(X=None, Y=None, C=None, metric='euclidean', step_sizes_sigma=None,
         Each row of the array contains an index pair n,m).
         Only returned when ``backtrack`` is True.
 
+    D_steps : np.ndarray [shape=(N,M)]
+        Step matrix, containg the indices of the used steps
+        Only returned when ``return_D_steps`` is True.
+
```

Comment:
is there any reason not to call this just `steps`?

Reply:
The variable was named `D_steps` before, so I just kept the name. In our group, we usually call it `E` (in order of computation: cost matrix `C`, accumulated cost matrix `D`, step size matrix `E`). However, I can also rename it to `steps` if you prefer this.

Reply:
I think `steps` is the clearest option here.  Single-letter names should be generally avoided.